app/graphql/overriden_ansible_variable_presenter.rb

```diff
@@ -0,0 +1,38 @@
+class OverridenAnsibleVariablePresenter
+  attr_reader :ansible_variable
+
+  delegate :id, :key, :description, :override?,
+           :parameter_type, :hidden_value?, :omit, :required,
+           :validator_type, :validator_rule, :default_value, :to => :ansible_variable
+
+  def initialize(ansible_variable, override_resolver)
+    @ansible_variable = ansible_variable
+    @override_resolver = override_resolver
+  end
+
+  def current_value
+    result = @override_resolver.resolve @ansible_variable
+    add_meta result
+  end
+
+  private
+
+  def known_type_class(element)
+    return unless element.is_a? String
+    return Operatingsystem if element == 'os'
+    element.classify.constantize rescue nil
```

Comment:
```suggestion
    element.classify.safe_constantize
```
Would also prevent the need to disable the rubocop